src/api/app/controllers/webui/monitor_controller.rb

```diff
@@ -74,8 +74,13 @@ def events
 
   private
 
+  UPPER_RANGE_LIMIT = 24 * 365
```

Comment:
```suggestion
  HOURS_IN_ONE_YEAR = 8760
```

It's a constant, it's always the same...

Reply:
True